torch/_dynamo/convert_frame.py

```diff
@@ -1159,6 +1161,8 @@ def clean_for_json(d: Dict[str, Any]) -> Dict[str, Any]:
                 "structured_logging_overhead_us": to_int_us(
                     structured_logging_overhead_s
                 ),
+                "cuda_version": torch.version.cuda,
```

Comment:
My comment was buried in another so maybe you missed it (or maybe I missed your response?. I think it makes more sense to put the logging here: https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/utils.py#L965

(this is common to multiple places where we do the logging, so if we put it here then all scuba logs will have these fields)

Reply:
Yeah @masnesral, I'll them here then.

Reply:
Added at correct location.